sass/DynamicHierarchicalFacet/_DynamicHierarchicalFacetValues.scss

```diff
@@ -110,6 +110,7 @@ $arrow-svg-dimensions: 11px;
 .coveo-dynamic-hierarchical-facet-value-arrow-left {
   position: absolute;
   left: 2px;
+  top: 0px;
```

Comment:
Looking at the mockup in the JIRA, the arrow is too far down. so your changes will overwrite the `top` property that is causing this issue, is this what's happening?

Reply:
Yes! since the top CSS property was not defined, IE11 was too naive to move it to the right spot, it was being affected by the other arrow